docs/fleet/_index.md

```diff
@@ -18,103 +18,62 @@ aliases:
 menuindent: true
 ---
 
-The [Viam app](https://app.viam.com) provides fleet management allowing you to work on any number of machines alone or in collaboration with others.
-You can manage and control your fleet of machines from the Viam app, using the [CLI](/fleet/cli/), or using the [cloud API](/build/program/apis/cloud/).
+Viam fleet management allows you to organize, manage, and control any number of machines alone or in collaboration with others.
+You can manage and control your fleet of {{< glossary_tooltip term_id="machine" text="smart machines" >}} from the Viam app, using the [CLI](/fleet/cli/), or using the [cloud API](/build/program/apis/cloud/).
 
 ## Work with groups of machines
 
-With Viam, you can organize {{< glossary_tooltip term_id="machine" text="smart machines" >}} into {{< glossary_tooltip term_id="location" text="locations" >}} and {{< glossary_tooltip term_id="organization" text="organizations" >}}.
+To organize your fleet you use
 
-For example, you may have separate organizations for your machines at home and at work:
+- {{< glossary_tooltip term_id="organization" text="organizations" >}}: the highest level grouping, generally used for different companies.
+- {{< glossary_tooltip term_id="location" text="locations" >}}: virtual groupings of devices up with up to three levels of nesting that can represent a grouping of machines that are co-located in a building, like a factory, or a grouping of machines that are thousands of miles apart and are grouped together by function or as an organizational unit.
 
-<!-- this is a very small gif - conversion to mp4 caused issues -->
-<img src="/manage/organizations.gif" alt="An organization for personal machines and one for work machines.">
+Each machine resides in a location.
 
-Inside an organization, you can organize machines into one or more locations:
+<!-- TODO: Add topology based on Jon's draft -->
```

Comment:
Will need to work with Cristina to get this stylized
![Screenshot 2024-02-02 at 19 13 25](https://github.com/viamrobotics/docs/assets/5212232/c81db353-e61f-4b0e-866c-6133cc047d4f)

Reply:
love it, was going to say exactly this and then saw your comment :) I can also help with that

Reply:
would you like it to be different from this? If so happy to have you work with Cristina and give us the result. If not I can also just create the ticket

Reply:
is your question whether you can reuse this specific diagram, or want to make your own, or something else? It does sound like there will be a design from Cristina immentently. Might as well reuse that? Let me know if you are asking something else